src/copy/_headers

```diff
@@ -8,3 +8,7 @@
 /*
   Cross-Origin-Embedder-Policy: require-corp
   Cross-Origin-Opener-Policy: same-origin
+# Origin trial for WebAssembly SIMD.
+# ATTENTION: This one is configured for dev--squoosh.netlify.app preview.
+# For production squoosh.app, replace with AgoKiDqjr0GVPtrwV/vuVlrrSvbDa5Yb99s+q66ly816DrrAQ8Cdas33NgDtmhxM4BtDP9PEdyuxHPyTQHD5ZAcAAABUeyJvcmlnaW4iOiJodHRwczovL3NxdW9vc2guYXBwOjQ0MyIsImZlYXR1cmUiOiJXZWJBc3NlbWJseVNpbWQiLCJleHBpcnkiOjE2MDg2NzI5OTR9.
```

Comment:
Might be time to finally switch this file to an .ejs file and use the right key depending on which branch is being built. But we can move this to a separate issue.

Reply:
Yeah Jake submitted one separately.